client/my-sites/github-deployments/connection-wizard-button.tsx

```diff
@@ -0,0 +1,87 @@
+import config from '@automattic/calypso-config';
+import page from '@automattic/calypso-router';
+import { Button } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { useState } from 'react';
+import SocialLogo from 'calypso/components/social-logo';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { useGithubAccountsQuery } from './use-github-accounts-query';
+
+const AUTHORIZE_URL = addQueryArgs( 'https://github.com/login/oauth/authorize', {
+	client_id: config( 'github_oauth_client_id' ),
+} );
+
+const POPUP_ID = 'github-oauth-authorize';
+
+const authorizeGitHub = () => {
+	return new Promise< void >( ( resolve, reject ) => {
+		const popup = window.open( AUTHORIZE_URL, POPUP_ID, 'height=600,width=700' );
```

Comment:
Do we want to avoid scrolls on that popup? I tested it with `height=960,width=800`, and it seems to work fine. Do we want to centralize it? I did a quick test with `const left = ( window.innerWidth - popupWidth ) / 2;` but it isn't perfect so that it would need more love.

Reply:
Could be. I just want to get it working in a good way. We can improve it in a follow-up.